index.html

```diff
@@ -17,6 +17,17 @@ <h1 class="primary-title">Front-End Tools</h1>
       <img src="assets/triangle.svg" class="img-title">
       <h1 class="primary-title">Back-End Tools</h1>
   </div>
+  <div class="tech-background size-tech">
+    <div class="tech size-tech">
+      <div class="order-tech size-tech modify">
+        <div><img src="assets/react_icon.svg" class="img-title"></div>
+        <div class="font-tech"> ReactJS</div>
```

Comment:
Don't use a <div> to format text, use h1, h2, etc for titles and <p> for paragraphs.
Same for the next lines :)